internal/commands/sbomtest/sbomtest.go

```diff
@@ -103,7 +103,7 @@ func TestWorkflow(
 
 	summaryData, summaryContentType, err := BuildTestSummary(results.Summary)
 	if err != nil {
-		return nil, errFactory.NewInternalError(err)
+		return nil, errFactory.NewInternalGenerationError(err)
```

Comment:
Should we also call this something `Fatal`, to be in line with the error from SBOM test?

Reply:
Yeah, it would make sense.